docs/overview.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: "Overview"
```

Comment:
3 of the core sections:

Software Carbon Intensity 
Water
Embodied emissions 

could probably be refactored to not be specifically GenAI overview, but a more general overview section